src/components/NetworkSelector/index.tsx

```diff
@@ -34,6 +34,11 @@ export const networkOptions: NetworkOptions[] = [
     name: 'Rinkeby',
     url: 'rinkeby',
   },
+  {
+    id: Network.GOERLI,
+    name: 'Goerli',
```

Comment:
It's German, so it needs the thingy on top of the `o`

```suggestion
    name: 'Görli',
```